dt-people-groups/people-groups-base.php

```diff
@@ -73,8 +73,24 @@ public function dt_details_additional_section( $section, $post_type ) {
     }
 
     public function dt_nav_filter( $navigation_array ) {
+        $is_hidden = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+
         if ( isset( $navigation_array['main'], $navigation_array['main'][ $this->post_type ] ) ) {
-            $navigation_array['main'][ $this->post_type ]['hidden'] = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+            $navigation_array['main'][ $this->post_type ]['hidden'] = $is_hidden;
+        }
+
+        if ( isset( $navigation_array['admin'], $navigation_array['admin']['add_new'] ) ) {
```

Comment:
@kodinkat what is happening here?

Reply:
Loops add-new submenu list and sets the people groups post type hidden flag accordingly, before updating the main navigation_array and returning.